### ISSUE TITLE
Impossible to use 'Search' since last update

### ISSUE THREAD

**Issue by fm4tt0s** [cars10/elasticvue#90]
- Elasticsearch version: 2.3
- Operating system: RHEL/running ES under docker
- Browser + version: Chrome Version 97.0.4692.99 (Official Build) (x86_64)
- Elasticvue version: 0.37.0
- How are you running elasticvue? : browser extension

When attempt to use 'Search' functionality it gives below error:

```
{
    "error": {
        "failed_shards": [
            {
                "index": "index_name",
                "node": "_xFRrOs-SgGU5-QTR3F8ew",
                "reason": {
                    "col": 69,
                    "line": 1,
                    "reason": "failed to parse search source. unknown search element [track_total_hits]",
                    "type": "search_parse_exception"
                },
                "shard": 0
            }
        ],
        "grouped": true,
        "phase": "query",
        "reason": "all shards failed",
        "root_cause": [
            {
                "col": 69,
                "line": 1,
                "reason": "failed to parse search source. unknown search element [track_total_hits]",
                "type": "search_parse_exception"
            }
        ],
        "type": "search_phase_execution_exception"
    },
    "status": 400
}
```

**Comment by cars10** [comment]
What happens when you customize the search query and remove `track_total_hits`?

![image](https://user-images.githubusercontent.com/4118628/152146879-4bd99ba6-cfbe-4b41-aa02-f33c85104fd3.png)
![image](https://user-images.githubusercontent.com/4118628/152146900-1ccc976d-4bcc-41d0-bec4-6ebb9605bee6.png)

**Comment by rodrigosouzazup** [comment]
> What happens when you customize the search query and remove `track_total_hits`?
> 
> ![image](https://user-images.githubusercontent.com/4118628/152146879-4bd99ba6-cfbe-4b41-aa02-f33c85104fd3.png) ![image](https://user-images.githubusercontent.com/4118628/152146900-1ccc976d-4bcc-41d0-bec4-6ebb9605bee6.png)

I have the same problem.  

When I remove this field the query is executed successfully.

**Comment by cars10** [comment]
Do you also use an outdated version of elasticsearch, or does this happen with a newer version?

**Comment by rodrigosouzazup** [comment]
We use AES with versions "7.10" and "5.6" and the problem occurs only in version "5.6"

**Comment by cars10** [comment]
#85 will probably be reverted.

**Comment by fm4tt0s** [comment]
@cars10 thanks for the attention on the matter. this is confirmed; removing `track_total_hits` from customized query payload make it work fine.

**Comment by cars10** [comment]
I have published `0.38.0` with some fixes for outdated versions of elasticvue. Please update your clusters! At least for security reasons.
